rbx_types/src/attributes/mod.rs

```diff
@@ -0,0 +1,141 @@
+use crate::variant::VariantType;
+use std::convert::TryFrom;
+use thiserror::Error;
+
+mod reader;
+mod writer;
+
+pub use reader::get_attributes;
+pub use writer::attributes_from_map;
+
+macro_rules! create_attribute_type {
+    ({
+        $(
+            $key:ident = $number:tt,
```

Comment:
```suggestion
            $key:ident = $number:literal,
```

We should prefer to use a more specific matcher than `tt` when possible. Here, `expr` or `literal` should fit well.